src/Raven.Server/Documents/Queries/QueryMetadata.cs

```diff
@@ -301,6 +305,21 @@ private void AddExistField(QueryFieldName fieldName, BlittableJsonReaderObject p
             }
 
             WhereFields[indexFieldName] = new WhereField(isFullTextSearch: search, isExactSearch: exact, spatial: spatial);
+
+            bool AllDocumentsSearch()
```

Comment:
What happens if this field does not exists?

Reply:
Cannot we just check in `QueryBuilder` inside `HandleSearch` method if the argument length is 1 and contains an asterisk? If yes we can run the same code as for `exists(field)`.

Reply:
Yes, that would be a better option, yes.